src/modules/75-cd/components/PipelineStudio/DeployStage/EditStageView/EditStageView.tsx

```diff
@@ -226,7 +226,7 @@ export const EditStageView: React.FC<EditStageViewProps> = ({
             }}
             validationSchema={Yup.object().shape(getNameAndIdentifierSchema(getString, contextType))}
           >
-            {formikProps => {
+            {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
            {(formikProps: unknown) => {
```